src/test/java/com/serenitydojo/WhenWorkingWithStrings.java

```diff
@@ -0,0 +1,44 @@
+package com.serenitydojo;
+
+
+import org.junit.Test;
+
+public class WhenWorkingWithStrings {
+
+    //Exercise 1 - Converting to lower case
+@Test
+    public void ConvertingToLowerCase(){
+        String myFirstName = "KINGSLEY";
+        System.out.println(myFirstName.toLowerCase());
+    }
+
+    //Exercise 2 - Converting to upper case
+
+@Test
+    public void ConvertingToUpperCase(){
+
+    String myFavoriteColor = "Green";
+    System.out.println(myFavoriteColor.toUpperCase());
+}
+
+// Exercise 3 - Trim extra space
+@Test
+    public void TrimExtraSpace(){
+    String myCountry = "Nigeria";
```

Comment:
that doesnt really show trimming. adding some space around the word would show it better
```suggestion
    String myCountry = "   Nigeria    ";
```

Reply:
Done